changelog/10.0.0_2024-07-29/change-remove-application-menu

```diff
@@ -0,0 +1,8 @@
+Change: Remove applicationMenu app property
+
+BREAKING CHANGE for developers: We've removed the `applicationMenu` app property in favour of the new `appMenuItem` extension type.
```

Comment:
We just deprecated the prop, does that mean we can actually remove it now? 🤔

Reply:
You removed all the handling in the runtime 🤔

Reply:
No I didn't 😅   I just ported our apps, but the fallback is still active: https://github.com/owncloud/web/blob/renovate/npm-run-all-replacement/packages/web-runtime/src/components/Topbar/TopBar.vue#L186

Reply:
ah... hm. sorry 😅 removed the changelog item again.